### ISSUE TITLE
BUG: Sorry, dbgate has crashed

### ISSUE THREAD

**Issue by orboan** [dbgate/dbgate#231]
**Describe the bug**
Using docker image, after creating a postgres connection (the only one I have at the moment) I click in "Application layers" icon and the browser says "Sorry, dbgate has crashed".

With docker logs containerid I get:
``` 
Loading module dbgate-plugin-postgres from /home/dbgate-docker/plugins/dbgate-plugin-postgres/dist/backend.js
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received null
    at new NodeError (internal/errors.js:322:7)
    at validateString (internal/validators.js:124:11)
    at Object.join (path.js:1148:7)
    at Object.files (/home/dbgate-docker/bundle.js:187:14854)
    at /home/dbgate-docker/bundle.js:348:2819
    at o.handle_request (/home/dbgate-docker/bundle.js:148:784)
    at o (/home/dbgate-docker/bundle.js:141:882)
    at u.dispatch (/home/dbgate-docker/bundle.js:141:904)
    at o.handle_request (/home/dbgate-docker/bundle.js:148:784)
    at /home/dbgate-docker/bundle.js:134:2535 {
  code: 'ERR_INVALID_ARG_TYPE'
```

**To Reproduce**
Execute docker-compose up -d on next docker-compose.yaml file:

```
version: '3.5'

services:
  postgres:
    container_name: postgres_container
    image: orboan/postgreshr:11.14
    environment:
      POSTGRES_USER: ${POSTGRES_USER:-postgres}
      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-admin}
      POSTGRES_HOST_AUTH_METHOD: trust
      PGDATA: /data/postgres
    volumes:
       - postgres:/data/postgres
    ports:
      - "5432:5432"
    networks:
      - postgres
    restart: unless-stopped
  pgadmin:
    container_name: pgadmin_container
    image: dpage/pgadmin4:6.5
    environment:
      PGADMIN_DEFAULT_EMAIL: ${PGADMIN_DEFAULT_EMAIL:-pgadmin4@pgadmin.org}
      PGADMIN_DEFAULT_PASSWORD: ${PGADMIN_DEFAULT_PASSWORD:-admin}
      PGADMIN_CONFIG_SERVER_MODE: 'False'
    volumes:
       - pgadmin_data:/var/lib/pgadmin
       - pgadmin_config:/pgadmin4
    ports:
      - "${PGADMIN_PORT:-5050}:80"
    networks:
      - postgres
    restart: unless-stopped
  dbgate:
    image: dbgate/dbgate
    restart: always
    ports:
      - 8081:3000
        #    environment:
    networks:
      - postgres
networks:
  postgres:
    driver: bridge

volumes:
    postgres:
    pgadmin_data:
    pgadmin_config:
```
Browsing to localhost:8081 with any browser the dbgate app works, but the error mentioned above.

**Expected behavior**
No error triggered and being able to navigate to the Application layers section of the app.

**Screenshots**

**Version Information (please complete the following information):**
 - OS: docker image run on ubuntu 20.04 as host
 - App Version: latest docker image
 - Install source: docker
 - Type - Web
 - Database engine: Postgres

**Additional context**
Anything else you think might be helpful

**Comment by janproch** [comment]
Fixed in current BETA

**Comment by orboan** [comment]
Thank you [janproch](https://github.com/janproch) for being so fast in fixing this. And thanks for all your effort to develop this very nice project / app.

**Comment by janproch** [comment]
Fixed in 4.7.1

Thanks for cooperation
